AdblockPlusSafari/AdblockPlusContainerController.m

```diff
@@ -40,22 +41,9 @@ - (void)viewDidLoad
 
   // This is the simpliest way to print Browser word using custom bold font.
   CGFloat fontSize = self.adblockBrowserLabel.font.pointSize;
-  UIFont *lightFont = [Appearence defaultLightFontOfSize:fontSize];
```

Comment:
It really seems like this is related to translations - as I said before, we should stick to #3093 in this pull request, it's big enough as it is.